threescale_api/resources.py

```diff
@@ -417,7 +417,7 @@ def service(self) -> 'Service':
     def list(self, **kwargs):
         if "env" in kwargs:
             self._env = kwargs["env"]
-            del(kwargs["env"])
+            del (kwargs["env"])
```

Comment:
this renaming doesn't seem to be relevant to the change in this PR, rather shouldn't be in this PR

Reply:
Fixed